pkg/mongo/service.go

```diff
@@ -23,7 +25,30 @@ func NewService(connectionURL string) (*Service, error) {
 		return nil, fmt.Errorf("could not dial mongo: %v", err)
 	}
 	db := client.Database("ekadashi")
-	return &Service{
+	s := &Service{
 		db: db,
-	}, nil
+	}
+	err = s.CreateIndex()
+	if err != nil{
+		return s, fmt.Errorf("cannot create an index: %v",err)
+	}
+	return s, nil
+}
+
+// CreateIndex creates an index for session collection
+func (s *Service) CreateIndex() error {
+	var opt options.IndexOptions
+	opt.SetExpireAfterSeconds(60*5)
+	opt.SetUnique(true)
+	keys := bsonx.Doc{{Key: "expiration", Value: bsonx.Int32(int32(1))}}
```

Comment:
Key is 'expiration', but TTL is set to 5 mins, which means object will be removed at expiration+5m. A bit confusing name for key then.

Reply:
can be simplified to `keys := bson.M{"expiration": 1}`